src/selectors/entities/users.ts

```diff
@@ -451,9 +454,9 @@ export function makeGetProfilesByIdsAndUsernames(): (a: GlobalState, b: {allUser
     return createSelector(
         getUsers,
         getUsersByUsername,
-        (state, props) => props.allUserIds,
+        (state: GlobalState, props: any) => props.allUserIds,
```

Comment:
Probably we can replace that `any` with an specific typing there (same in the the next line)